### DISCUSSION TITLE
Contribution graph color pallets

### DISCUSSION THREAD

**Discussion by sudojunior** [community/community#7078] [Other Feature Feedback, Questions, & Ideas] [answered]
Is the change in the contribution graph colors *seasonal* or would we get the option to select our own in the future?
*If the latter*, would this allow custom colour pallets or a select few for accessibility reasons?

## Current colour pallets

> The tables below have links directed towards `https://coolors.co/*` for both color pallets and individual color details.

<sup>*1</sup> Also referred to as `bg`, but attributed using `data-level="0"` on elements.
<sup>*2</sup> Listed as CSS variable `--color-scale-grey-8`, and also not available to the Browser Inspector except as its assigned color.
<sup>*3</sup> The Radar Graph mapping contribution ratios uses **Level 4**.

- **Default light** also covers **Light colorblind**
- **Default dark** also covers **Dark dimmed**, **Dark high contrast** and **Dark colorblind**

### Regular / Default

> CSS variable for Levels 1 through 4: `--color-calendar-graph-day-L{n}-bg`
> CSS variable for the boarder of each level, but remains unchanged at this time: `---color-calendar-graph-day-L{n}-boarder`

| Type        | [Light Mode](https://coolors.co/ebedf0-9be9a8-30c463-30a14e-261e39) | [Dark Mode](https://coolors.co/161b22-0e4429-006d32-26a641-39d353) |
| ----------- | ------------ | ----------- |
| Level 0 <sup>*1</sup> | [`#ebedf0`](https://coolors.co/ebedf0) | [`#161b22`](https://coolors.co/161b22) <sup>*2</sup> |
| Level 1     | [`#9be9a8`](https://coolors.co/9be9a8) | [`#0e4429`](https://coolors.co/0e4429) |
| Level 2     | [`#30c463`](https://coolors.co/30c463) | [`#006d32`](https://coolors.co/006d32) |
| Level 3     | [`#30a14e`](https://coolors.co/30a14e) | [`#26a641`](https://coolors.co/26a641) |
| Level 4 <sup>*3</sup> | [`#216e39`](https://coolors.co/216e39) | [`#39d353`](https://coolors.co/39d353) |

### Halloween

> CSS Variable for Levels 1 through 4: `--color-calendar-halloween-graph-day-L{n}-bg`

| Type        | [Light Mode](https://coolors.co/ebedf0-ffee4a-ffc501-de9600-03001c) | [Dark Mode](https://coolors.co/161b22-631c03-bd561d-fa7a18-fddf68) |
| ----------- | ---------- | --------- |
| Level 0 <sup>*1</sup> | [`#ebedf0`](https://coolors.co/ebedf0) | [`#161b22`](https://coolors.co/161b22) <sup>*2</sup> |
| Level 1     | [`#ffee4a`](https://coolors.co/ffee4a) | [`#631c03`](https://coolors.co/631c03) |
| Level 2     | [`#ffc501`](https://coolors.co/ffc501) | [`#bd561d`](https://coolors.co/bd561d) |
| Level 3     | [`#fe9600`](https://coolors.co/fe9600) | [`#fa7a18`](https://coolors.co/fa7a18) |
| Level 4 <sup>*3</sup> | [`#03001c`](https://coolors.co/03001c) | [`#fddf68`](https://coolors.co/fddf68) |

### Winter

| Type        | [Light Mode](https://coolors.co/ebedf0-b6e3ff-54aeff-0969da-0a3069) | [Dark Mode](https://coolors.co/161b22-0a3069-0969da-54aeff-b6e3ff) |
| ----------- | ---------- | --------- |
| Level 0 <sup>*1</sup> | [`#ebedf0`](https://coolors.co/ebedf0) | [`#161b22`](https://coolors.co/161b22) <sup>*2</sup> |
| Level 1     | [`#b6e3ff`](https://coolors.co/b6e3ff) | [`#0a3069`](https://coolors.co/0a3069) |
| Level 2     | [`#54aeff`](https://coolors.co/54aeff) | [`#0969da`](https://coolors.co/0969da) |
| Level 3     | [`#0969da`](https://coolors.co/0969da) | [`#54aeff`](https://coolors.co/54aeff) |
| Level 4 <sup>*3</sup> | [`#0a3069`](https://coolors.co/0a3069) | [`#b6e3ff`](https://coolors.co/b6e3ff) |

**Comment by gaknoll** [comment, marked as answer]
@sudojunior the change in colors was just for the day of Halloween! We do not have plans at this time to allow users to choose a custom color palette for the contribution graph
